### PR TITLE
Add R9 Mini to R9MM section

### PR DESCRIPTION
**Pull request by jlpoltrack**
Confirmed by Geo that R9 Mini is functionally the same as R9MM.   https://www.rcgroups.com/forums/showpost.php?p=51754991&postcount=2173

### CODE REVIEW

**Comment by olliw42** [comment]
cool
isn't it called R9Mini? I.e. maybe R9MM and R9Mini instad of R9MM/Mini. Or R9MM/R9Mini

**Comment by jlpoltrack** [comment]
Update to 'R9 MX', 'R9 MM' and 'R9 Mini' to match Frsky product pages.

image

image

File: docs/FRSKY_R9.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# mLRS Documentation: Frsky R9M, R9M Lite Pro, R9MX #
+# mLRS Documentation: Frsky R9M, R9M Lite Pro, R9 MX #
 
 ([back to main page](../README.md))
 
-The Frsky R9M, R9M Lite Pro transmitter modules and R9MX, R9MM receivers can be a relatively easy way to get started with mLRS as they are commercially available. However, the R9 hardware does have some limitations as detailed below which may affect some applications.
+The Frsky R9M, R9M Lite Pro transmitter modules and R9 MX, R9MM receivers can be a relatively easy way to get started with mLRS as they are commercially available. However, the R9 hardware does have some limitations as detailed below which may affect some applications.
```
Comment (olliw42) [inline docs/FRSKY_R9.md:5]: R9 MM
shouldn't we maybe list the R9 Mini here too?

File: docs/FRSKY_R9.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ The Frsky R9M, R9M Lite Pro transmitter modules and R9MX, R9MM receivers can be
     </tr>
     <tr>
       <td>Max. RF Output Power</td>
-      <td>30 dBm (1 W) for R9M, R9M Lite Pro, 17 dBm (50 mW) for R9MX, R9MM</td>
+      <td>30 dBm (1 W) for R9M, R9M Lite Pro, 17 dBm (50 mW) for R9 MX, R9MM</td>
```
Comment (olliw42) [inline docs/FRSKY_R9.md:15]: R9 MX, R9 MM, R9 Mini ?

File: docs/FRSKY_R9.md [hunk 3]
```diff
@@ -273,11 +273,11 @@ Note: In the ExpressLRS docs it is suggested to download and use the "ST-LINK Ut
...
 ### Update via System Bootloader ###
 
-When the R9MX or R9MM receivers have been flashed with the non-"elrs-bl" firmware version via ST-Link, they can be upgraded by invoking the STM32 system bootloader, which can be done as follows:
+When the R9 MX / R9 MM / R9 Mini receivers have been flashed with the non-"elrs-bl" firmware version via ST-Link, they can be upgraded by invoking the STM32 system bootloader, which can be done as follows:
 
 - Download and install STM32CubeProgrammer
-- Connect the R9MX/R9MM via the serial Rx/Tx pins to a USB-TTL adpater
-- Power up the R9MX/R9MM while keeping the button pressed; this boots the receiver into the system bootloader
+- Connect the R9 MX / R9 MM / R9 Mini via the serial Rx/Tx pins to a USB-TTL adpater
+- Power up the R9 MX / R9MM / R9 Mini while keeping the button pressed; this boots the receiver into the system bootloader
```
Comment (olliw42) [inline docs/FRSKY_R9.md:280]: R9 MM

**Comment by olliw42** [comment]
interesting
I admit when I first saw this change I kind of didn't liked it since I believe that R9MM and R9MX is kind of used "everywhere" ... except as you point out on FrSky's pages ... and we do have kind of the rule to follow the originator's convention => good catch :) :D

is true howver also for R9 MM: https://www.frsky-rc.com/product/r9-mm/ :) not yet done consistently

File: docs/FRSKY_R9.md (hunk 1 above)
Comment (jlpoltrack) [reply docs/FRSKY_R9.md:5]: Added

File: docs/FRSKY_R9.md (hunk 2 above)
Comment (jlpoltrack) [reply docs/FRSKY_R9.md:15]: Added

File: docs/FRSKY_R9.md (hunk 3 above)
Comment (jlpoltrack) [reply docs/FRSKY_R9.md:280]: Fixed

**Comment by jlpoltrack** [comment]
Should be consistent now - addressed all of your comments :)

**Comment by olliw42** [comment]
lgtm
MANY THX

**Pull request merged**